drivers/hmis/app/models/hmis/form/definition.rb

```diff
@@ -66,6 +67,7 @@ class Hmis::Form::Definition < ::GrdaWarehouseBase
   ].freeze
 
   validates :role, inclusion: { in: FORM_ROLES.map(&:to_s) }
+  validates :identifier, uniqueness: true
```

Comment:
Can we switch this to validate uniqueness of identifier+version? As discussed in the future there will be ability to create new versions of the "same" definition